controller/types/types.go

```diff
@@ -15,12 +15,13 @@ import (
 const RouteParentRefPrefix = "controller/apps/"
 
 type ExpandedFormation struct {
-	App       *App                         `json:"app,omitempty"`
-	Release   *Release                     `json:"release,omitempty"`
-	Artifact  *Artifact                    `json:"artifact,omitempty"`
-	Processes map[string]int               `json:"processes,omitempty"`
-	Tags      map[string]map[string]string `json:"tags,omitempty"`
-	UpdatedAt time.Time                    `json:"updated_at,omitempty"`
+	App           *App                         `json:"app,omitempty"`
+	Release       *Release                     `json:"release,omitempty"`
+	ImageArtifact *Artifact                    `json:"artifact,omitempty"`
```

Comment:
The dashboard shouldn't be effected if the JSON field name stays the same (otherwise there's a [single reference](https://github.com/flynn/flynn/blob/master/dashboard/app/lib/javascripts/dashboard/views/app-history.js.jsx#L73) that would need updating using the new name).